site2/website/versioned_sidebars/version-2.8.3-sidebars.json

```diff
@@ -384,7 +384,7 @@
         },
         {
           "type": "doc",
-          "id": "version-2.8.2/security-basic-auth"
+          "id": "version-2.8.3/security-basic-auth"
```

Comment:
reminder: any other doc versions need this chapter?

Reply:
This is a fix. Other doc versions require this chapter have been implemented in #15734.